en/sim_gazebo_gz/index.md

```diff
@@ -174,6 +174,46 @@ This can be ignored:
 
 :::
 
+### Change Simulation Speed
```

Comment:
@dirksavage88 @dakejahl Given that jMAVSim is "community supported" and the implementations of the three sims are different I have spawned "Change Simulation Speed" into all three docs. The top level simulation page is now very thin pointers to the information in specific simulator pages.

Can you please sanity check the changes? Also to make sure this is inline with the update PR that @dakejahl did.